app/src/controllers/initialData/sagas.js

```diff
@@ -49,6 +54,8 @@ function* fetchInitialData() {
     yield put(authSuccessAction());
   } else {
     yield put(resetTokenAction());
+    yield put(resetPluginAction());
```

Comment:
Looks like the `resetPluginAction` is not necessary as you will define plugins to select based on action type.